.github/workflows/build.yml

```
@@ -0,0 +1,27 @@
+name: Build Docker Package
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build and push xpub-scan image
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      - name: Clone
+        uses: actions/checkout@v2
+
+      - name: Build and push xpub-scan to GitHub Packages
+        uses: docker/build-push-action@v1
+        with:
+          file: Dockerfile
+          repository: ledgerhq/xpub-scan
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+          tag_with_sha: true
+          tags: latest
```

In order to push to docker.pkg.github.com (instead of ghcr, which may be preferable?), here's a proposition to make it work. 
I also proposed a change to the push from master to push from tags. It helps with version handling.

```suggestion
name: Build Docker Package
on:
  push:
    tags:
      - '*'

jobs:
  build:
    name: Build and push xpub-scan image
    runs-on: ubuntu-latest
    permissions:
      packages: write

    steps:
      - name: Clone
        uses: actions/checkout@v2
        
      - name: Build and push xpub-scan to GitHub Packages
        uses: docker/build-push-action@v2
        with:
          file: Dockerfile
          repository: ledgerhq/xpub-scan
          registry: docker.pkg.github.com
          username: ${{ secrets.CI_BOT_USERNAME }}
          password: ${{ secrets.CI_BOT_TOKEN }}
          tags: latest
          tag_with_ref: true
          add_git_labels: true
          push: ${{ startsWith(github.ref, 'refs/tags/') }}
```

---

docker.pkg.github.com is in the process of being deprecated